JASP-Desktop/data/datasetpackage.h

```diff
@@ -39,7 +41,14 @@ class EngineSync;
 class DataSetPackage : public QAbstractItemModel //Not QAbstractTableModel because of: https://stackoverflow.com/a/38999940 (And this being a tree model)
 {
 	Q_OBJECT
-	Q_PROPERTY(int columnsFilteredCount READ columnsFilteredCount		NOTIFY columnsFilteredCountChanged)
+	Q_PROPERTY(int			columnsFilteredCount	READ columnsFilteredCount							NOTIFY columnsFilteredCountChanged	)
+	Q_PROPERTY(QString		name					READ name											NOTIFY nameChanged					)
+	Q_PROPERTY(QString		folder					READ folder					WRITE setFolder			NOTIFY folderChanged				)
+	Q_PROPERTY(QString		windowTitle				READ windowTitle									NOTIFY windowTitleChanged			)
+	Q_PROPERTY(bool			modified				READ isModified				WRITE setModified		NOTIFY isModifiedChanged			)
+	Q_PROPERTY(bool			loaded					READ isLoaded				WRITE setLoaded			NOTIFY loadedChanged				)
+	Q_PROPERTY(QFileInfo	currentFile				READ currentFile									NOTIFY currentFileChanged			)
+	Q_PROPERTY(QString		currentPath				READ currentPath			WRITE setCurrentPath	NOTIFY currentPathChanged			)
```

Comment:
I'm not sure what is the difference between folder and currentPath, and whether it is necessary to have 2 properties for this. Or could you give a better name?

Reply:
currentPath is the path to where the file is stored. I guess it could be merged back into curentFile. But at some point I thought I needed a QUrl for OSF files (currentPath) and a QFileInfo for normal files (currentFile).

Ill merge them!

Reply:
Done